scipy/spatial/transform/rotation.py

```diff
@@ -1465,6 +1466,60 @@ def magnitude(self):
         else:
             return angles
 
+    def mean(self, weights=None):
+        """Get the mean of the rotation(s).
+
+        Parameters
+        ----------
+        weights : array_like shape (N,), optional
+            Weights describing the relative importance of the rotations. If
+            None (default), then all values in `weights` are assumed to be
+            equal.
+
+        Returns
+        -------
+        mean : `Rotation` instance
+            Object containing the mean of the rotations in the current
+            instance.
+
+        Notes
+        -----
+        The mean used is the chordal L2 mean (also called the projected or
+        induced arithmetic mean). If `p` is a set of rotations with mean `m`,
+        then `m` is the rotation which minimizes
+        `((p.as_dcm() - m.as_dcm())**2).sum()`.
+
+        Examples
+        --------
+        >>> from scipy.spatial.transform import Rotation as R
+        >>> r = R.from_euler('zyx', [
+        ... [0, 0, 0],
```

Comment:
Alignment is not very good, try formatting first in an editor and then add >>> and ...

Reply:
I think it is better now.